app/components/Admin/ProposedService.js

```diff
@@ -86,6 +95,16 @@ class ProposedService extends React.Component {
     }
     return scheduleOutput;
   }
+
+  renderCategoryFields() {
+    return (
+      <div key={"cat"} className="request-entry">
```

Comment:
I don't really know React that well, but do you need a `key` if this isn't really a list of stuff?